tests/Tests/600__ai_apps/680__starburst/680__starburst.robot

```diff
@@ -0,0 +1,41 @@
+*** Settings ***
+Library             SeleniumLibrary
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+
+Test Setup          Dashboard Test Setup
+Test Teardown       Dashboard Test Teardown
+
+
+*** Variables ***
+${SB_CARDS_XP}=     //article[@id="starburst"]
+${SB_BETA_DESC}=    //*[@class="pf-c-drawer__panel-main"]//div[@class="pf-c-alert pf-m-inline pf-m-info"]
+
+
+*** Test Cases ***
+Verify if the Starburst Beta text has been removed from Getting Started
+    [Tags]    ODS-1158    ODS-605
+
+    Click Link    Explore
+    Wait Until Cards Are Loaded
+    Open Get Started Sidebar And Return Status    card_locator=${SB_CARDS_XP}
+    Check Beta Description
+
+
+*** Keywords ***
+Check Beta Description
+    ${version-check}=    Is RHODS Version Greater Or Equal Than    1.8.0
+    IF    ${version-check}==True
+        Page Should not Contain Element    xpath:${SB_BETA_DESC}
+    ELSE
+        Page Should Contain Element    xpath:${SB_BETA_DESC}
+    END
```

Comment:
Is this supposed to be 1.8 or did you mean 1.7?

Reply:
1.8